### PR TITLE
Added portuguese (pt_BR) Adobe Reader

### PR DESCRIPTION
**Pull request by alexandregz**
Adobe only have brasilian version

### CODE REVIEW

File: Casks/adobe-reader-pt.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask :v1 => 'adobe-reader-pt' do
+  version '11.0.00'
+  sha256 '432b83f2d895c42c496226214e5b700e1ee078bc03b4501500b9a783d63e4dad'
+
+  url "http://ardownload.adobe.com/pub/adobe/reader/mac/#{version.to_i}.x/#{version}/pt_BR/AdbeRdr#{version.gsub('.', '')}_pt_BR.dmg"
+  homepage 'http://www.adobe.com/br/products/reader.html'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/adobe-reader-pt.rb]: Should be `:gratis`.

**Comment by vitorgalvao** [comment]
Please add `name` and `vendor`. Follow [the main cask](https://github.com/caskroom/homebrew-cask/blob/master/Casks/adobe-reader.rb).

**Comment by alexandregz** [comment]
rebased too

**Comment by alexandregz** [comment]
And thx for you patience!

**Comment by alexandregz** [comment]
and done here too

**Pull request merged**